### PR TITLE
Use the adafruit cc3000 module for wifi gateway

### PR DESCRIPTION
**Pull request by PressY4EKG**
i had previously started this, but never even realized the development branch existed, and i never bothered to recomit my work to master. 

You can ignore my myconfig.h i forgot to ignore it when i commited

### CODE REVIEW

**Comment by fallberg** [comment]
Jenkins, build this please

File: libraries/MySensors/examples/GatewayCC3000/GatewayCC3000.ino [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+ * - ERR (red) - fast blink on error during transmission error or recieve crc error 
+ * 
+ */
+
+// Enable debug prints to serial monitor
+#define MY_DEBUG 
+
+#define MY_RF24_CE_PIN 4
+#define MY_RF24_CS_PIN 5
+#define MY_RF24_CHANNEL  111
```
Comment (henrikekblad) [inline libraries/MySensors/examples/GatewayCC3000/GatewayCC3000.ino]: Should it really set 111 here ?

**Comment by fallberg** [comment]
Jenkins, build this please

File: libraries/MySensors/examples/GatewayCC3000/GatewayCC3000.ino (hunk 1 above)
Comment (PressY4EKG) [inline libraries/MySensors/examples/GatewayCC3000/GatewayCC3000.ino]: no it shouldnt sorry. I just have a set up on 76 and didnt want to get in the way with my other network

File: libraries/MySensors/MyConfig.h [hunk 2]
```diff
@@ -172,7 +172,7 @@
 // but now can be used in any sensor node. Also the LEDs can now be
 // disabled in the gateway.
 
-//#define MY_LEDS_BLINKING_FEATURE
+#define MY_LEDS_BLINKING_FEATURE
```
Comment (henrikekblad) [inline libraries/MySensors/MyConfig.h]: You should probably leave MyConfig.h unchanged as well. 

Please update PR.

**Comment by henrikekblad** [comment]
Thanks, 

Would you mind removing the MyHwCC3000.cpp and MyHwCC3000.h as well.

**Comment by henrikekblad** [comment]
Or rather start from a fresh PR from a clean checkout of development. I see there are a few weird merges/commits.  Nice to keep git history clean.

**Pull request closed without merging**